src/elements/calendar/DayOfMonth.pipe.ts

```diff
@@ -0,0 +1,9 @@
+import { Pipe, PipeTransform, LOCALE_ID, Inject } from '@angular/core';
+
+@Pipe({ name: 'dayofmonth' })
+export class DayOfMonthPipe implements PipeTransform {
+    constructor(@Inject(LOCALE_ID) private locale: string = 'en-US') {}
```

Comment:
How will this `LOCALE_ID` get changed for apps that can override the browser lang?